_config.yml

```diff
@@ -1,9 +1,9 @@
 # standard jekyll configuration
 
-title_en: "Documentation - Web Experience Toolkit&#160;(WET)"
-title_fr: "Documentation - Boîte à outils de l'expérience Web&#160;(BOEW)"
-description_en: "Documentation for the Web Experience Toolkit (WET)."
-description_fr: "Documentation pour la Boîte à outils de l'expérience Web (BOEW)."
+title_en: "Web Experience Toolkit (WET)"
+title_fr: "Boîte à outils de l'expérience Web (BOEW)"
+description_en: "Documentation for Web Experience Toolkit (WET)"
+description_fr: "Documentation for Web Experience Toolkit (WET)"
```

Comment:
Don't change this line

Reply:
Can you elvabourate as to why?

I as I stated above I have not yet done testing, but I converted the style guide to the same codec (GitHub pages) and this file is needed to be updated for my changes to for.

I do not speak French so if you can provide French translation I can update that line.

Reply:
There was already a suitable French translation in the line you replaced

Reply:
Did not notice. Thanks.